zsh-bash-completions-fallback.plugin.zsh

```diff
@@ -91,7 +97,17 @@ function _bash_completions_fetch_supported_commands {
 }
 
 function _bash_completions_load {
-    local bash_completions=${ZSH_BASH_COMPLETIONS_FALLBACK_PATH:-${${(@s/:/)${XDG_DATA_DIRS:-/usr/share}}[1]}/bash-completion}
+    local dir bash_completions
+    if [ -d "$ZSH_BASH_COMPLETIONS_FALLBACK_PATH" ]; then
+        bash_completions=$ZSH_BASH_COMPLETIONS_FALLBACK_PATH
+    else
+        for dir in "${_bash_completions_dirs[@]}"; do
+            if [ -d $dir/bash-completion ]; then
```

Comment:
Also break only if `[ -f "$dir/bash-completion/bash_completion" ]`

Reply:
Even if we remove `$bash_completions`, the plugin still can work.

Reply:
See <https://github.com/3v1n0/zsh-bash-completions-fallback/pull/12/commits/d9ad97999eab26bb7b3851af407f03762f68ffee>